src-java/rule-manager/rule-manager-api/src/main/java/org/openkilda/group/Bucket.java

```diff
@@ -0,0 +1,34 @@
+/* Copyright 2021 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.group;
+
+import org.openkilda.action.Action;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.Builder;
+import lombok.Value;
+
+import java.util.List;
+
+@Value
+@JsonSerialize
+@Builder
+public class Bucket {
+
+    WatchGroup watchGroup;
+    WatchPort watchPort;
+    List<Action> actions;
```

Comment:
It is ActionSet, so we should try to represent it as Set. To show everyone that they can't control actions order here.

Reply:
Fixed.